ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -3676,6 +3676,47 @@ def test_torch_instance_cumsum(
     )
 
 
+# cumsum_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="cumsum_",
+    dtype_value=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
```

Comment:
seems like the ground truth does not support `bool`, you can change "valid" to "numeric" here